symbolic_full_path

```diff
@@ -0,0 +1 @@
+/Users/evan/Documents/Core-v4/util/mock-database.js
```

Comment:
Seems like a local file... Maybe gitignore it or use a relative path?

Reply:
thank u for pointing this out!! removing it now.